crates/swc_css_codegen/tests/fixture/at-rules/import/1/output.min.css

```diff
@@ -0,0 +1 @@
+@import'custom.css';@import url("chrome://communicator/skin/");@import url("fineprint.css") print;@import url("bluish.css") speech;@import"common.css" screen;@import url('landscape.css') screen and (orientation:landscape);
```

Comment:
I tested using w3c css validator, and semes like the space after `)` is not required.

It's fine if it's intentional, I'm asking just to ensure it's not a mistake.

Reply:
Do you mean `@import url('landscape.css') screen`? Yes, also space is not required after `@import"common.css"screen;`

I will improve

Reply:
@kdy1 Done :+1: